samples/bluetooth/direct_test_mode/README.rst

```diff
@@ -69,20 +70,35 @@ For this reason, a coaxial connection between the DUT and the Lower Tester is em
 DTM module interface
 ====================
 
-The DTM function ``dtm_cmd_put`` implements the four commands defined by the Bluetooth Low Energy standard:
+The DTM module interface can be divided into following three parts:
 
-* ``TEST SETUP`` (called ``RESET`` in Bluetooth 4.0)
-* ``RECEIVER_TEST``
-* ``TRANSMITTER_TEST``
-* ``TEST_END``
+* The :c:func:`dtm_init` initialization function
+* Setup functions:
 
-In the ``dtm_cmd_put`` interface, DTM commands are accepted in the 2-byte format.
-Parameters such as ``CMD code``, ``Frequency``, ``Length``, or ``Packet Type`` are encoded within this command.
+  * :c:func:`dtm_setup_reset`
+  * :c:func:`dtm_setup_set_phy`
+  * :c:func:`dtm_setup_set_modulation`
+  * :c:func:`dtm_setup_read_features`
+  * :c:func:`dtm_setup_read_max_supported_value`
+  * :c:func:`dtm_setup_set_cte_mode`
+  * :c:func:`dtm_setup_set_cte_slot`
+  * :c:func:`dtm_setup_set_antenna_params`
+  * :c:func:`dtm_setup_set_transmit_power`
 
-The following DTM events are polled using the ``dtm_event_get`` function:
+* Test functions:
 
-* ``PACKET_REPORTING_EVENT``
-* ``TEST_STATUS_EVENT`` [ ``SUCCESS`` | ``FAIL`` ]
+  * :c:func:`dtm_test_receive`
+  * :c:func:`dtm_test_transmit`
+  * :c:func:`dtm_test_end`
+
+The setup functions implement the DTM setup required by the Bluetooth Low Energy standard.
+
```

Comment:
I recommend removing this empty line to have both similar sentences in one paragraph.